src/measure-test-fixed-encoding/dune

```diff
@@ -1,4 +1,3 @@
 (executable
  (name MeasureTestFixedEncoding)
- (public_name MeasureTestFixedEncoding)
```

Comment:
Why was this removed?

Reply:
Because it is used only during the dev time. We can still exec it see https://github.com/jordwalke/flex/blob/master/package.json#L39. With this change dune need to build 138 instead of 150 targets. It save the 3Mb of this exe.